UnleashedRecomp/user/config.cpp

```diff
@@ -3,6 +3,10 @@
 #include <ui/game_window.h>
 #include <user/paths.h>
 
+#if defined(__linux__)
+    const bool g_isRunningUnderFlatpak = std::filesystem::exists("/.flatpak-info");
```

Comment:
In that case, use the std::error_code variant for this, or it can throw an exception on failure and just crash the app. 
You can use a lambda inplace and immediately call it like so:
```suggestion
    const bool g_isRunningUnderFlatpak = []()
    { 
        std::error_code ec;
        return std::filesystem::exists("/.flatpak-info", ec);
    }();
```